### PR TITLE
test(fix): update the test proxy to set app profile id and connect to emulator correctly

### PR DESCRIPTION
**Pull request by liujiongxin**
This change makes the test proxy work with the hermetic testing that exercises app_profile_id and emulator (https://github.com/googleapis/cloud-bigtable-clients-test)

### CODE REVIEW

File: test-proxy/src/main/java/com/google/cloud/bigtable/testproxy/CbtTestProxy.java [hunk 1]
```diff
@@ -215,7 +215,11 @@ public synchronized void createClient(
       return;
     }
 
-    BigtableDataSettings.Builder settingsBuilder = BigtableDataSettings.newBuilder();
+    BigtableDataSettings.Builder settingsBuilder = BigtableDataSettings.newBuilder()
+        .setProjectId(request.getProjectId())
+        .setInstanceId(request.getInstanceId())
+        .setAppProfileId(request.getAppProfileId());
```
Comment (mutianf) [inline test-proxy/src/main/java/com/google/cloud/bigtable/testproxy/CbtTestProxy.java:221]: what if the request doesn't have an app profile id?

**Review by mutianf — APPROVED** [review]

**Comment by mutianf** [comment]
Please update the commit message and the name of the pr to be:

fix: Update the test proxy to accommodate custom app profile and emulator #1529

**Comment by kolea2** [comment]
> Please update the commit message and the name of the pr to be:
> 
> fix: Update the test proxy to accommodate custom app profile and emulator #1529

Can we prefix these as `test(fix)` to skip them in the release notes?

File: test-proxy/src/main/java/com/google/cloud/bigtable/testproxy/CbtTestProxy.java (hunk 1 above)
Comment (liujiongxin) [reply test-proxy/src/main/java/com/google/cloud/bigtable/testproxy/CbtTestProxy.java:221]: I also add the prefix "test(fix)" to the title and commit.

**Pull request merged**